### PR TITLE
Create angular2.yml

### PR DESCRIPTION
**Pull request by PatrickJS**

### CODE REVIEW

File: _data/projects/angular2.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Angular2
+desc: HTML enhanced for web apps
+site: https://angular.io
+tags:
+- JavaScript
+- web
+- Angular 2
+upforgrabs:
+  name: hotlist: community-help
```
Comment (shiftkey) [inline _data/projects/angular2.yml]: The `:` here is causing a parsing issue. You'll need to wrap this value in double-quotes.

**Comment by shiftkey** [comment]
Apologies for the delay on reviewing this. I found a parsing issue which will mean the project won't appear correctly on the site if this was merged.

Aside from that this is good to go!

**Comment by PatrickJS** [comment]
@shiftkey done()

**Comment by shiftkey** [comment]
Thanks!

**Pull request merged**